src/components/combobox/utils.ts

```diff
@@ -39,3 +40,7 @@ export function getDepth(element: HTMLElement): number {
 
   return result.snapshotLength;
 }
+
+export async function scrollTo(x: number, y: number, page: E2EPage): Promise<void> {
```

Comment:
Is this used? I think I had suggested possibly creating a reusable function for tests, not the component. If not needed at the moment, let's remove it.